app/validations/site-validation.js

```diff
@@ -0,0 +1,73 @@
+import Joi from 'joi';
+
+const belgiumPhoneNumberRegex = /^(?:\+32|0)[4-9][0-9]{8}$/;
+const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+const websiteRegex = /^https:\/\//;
```

Comment:
This does not cover http and it does not check for the minimum necassary url components.
In fact. The http/https part is optional in this context. At the very leasy you need a top level (.com, .be) and a second level (bol, amazon, whatever). Some non technical users will actually freak out if they have to type https or something.
This is a valid url: "vlaanderen.be" which is transformed into "https://vlaanderen.be"
This is also a valid url: "http://dennis:password@www.vlaanderen.be:8300/derp?parameter=lol#first-section"
**Do not make this regex yourself but google for it.** I found this one:
`https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)`

Reply:
Also i found this https://regexr.com/39nr7